test/specs/SellecktPopup.specs.js

```diff
@@ -579,6 +579,25 @@ function sellecktPopupSpecs(SellecktPopup, templateUtils, $, _, Mustache){
                 expect(focusStub.thisValues[0].is(popup.$popup.find('.' + popup.itemClass).eq(0))).toEqual(true);
             });
 
+            it('focuses the first item if $input is undefined', function(){
+                var focusStub = sandbox.stub($.fn, 'focus');
+                var ITEMS_CONTAINER = '<div class="{{itemsClass}}">' +
+                    '<ul class="{{itemslistClass}}">' +
+                        '{{#items}}' +
+                            '{{> item}}' +
+                        '{{/items}}' +
+                    '</ul>' +
+                '</div>';
+
+                popup = new SellecktPopup({
+                    template: ITEMS_CONTAINER,
+                    showSearch: true
+                });
+                popup.open($opener, items);
+
+                expect(focusStub.calledOnce).toEqual(true);
```

Comment:
Yes, thanks @allouis!
I had tested this, I don't know how this line disappeared. Now it's back again 👍